open_mastr/utils/helpers.py

```diff
@@ -249,6 +250,10 @@ def raise_warning_for_invalid_parameter_combinations(
             "(with prefix bulk_) are ignored."
         )
 
+
+def db_engine():  # TODO: Include in _create_database in MaStR() class
+    return create_engine(DB_URL)
```

Comment:
@deniztepe If I remember correct you deleted that function, or am I wrong?

Reply:
Yes, it seems like that these 3 lines are slipped in during a merge.

Reply:
I deleted it again